<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>10</number>
   </property>
   <item row="0" column="0" colspan="4">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame
{
	background-color: rgb(94, 84, 115);
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
	background-color: rgba(255, 255, 255, 0);
	color:rgb(220,212,229);
}</string>
        </property>
        <property name="text">
         <string>Мои курсы</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="settingsButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background-color: rgba(0,0,0, 0);
	image: url(icons/settings);
	border: none;
	color: rgb(40,40,40);
	padding: 7 7 7 7;
	border-radius: 20px;

}
	QPushButton:hover
	{
		padding: 6 6 6 6;
	}
	QPushButton:pressed
	{
		background-color: rgba(0,0,0, 40);
		padding: 4 4 4 4;	
	}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0">
    <layout class="QVBoxLayout" name="ListLay">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>26</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="assignTab">
        <property name="geometry">
         <rect>
          <x>546</x>
          <y>0</y>
          <width>180</width>
          <height>26</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>26</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background-color: rgb(230, 230, 230);
	border: none;
	color: rgb(40,40,40);
	text-align: left;
	padding-left: 6px;
	padding-top:2px;
	margin-top:6;
}
	QPushButton:hover
	{
		margin-top:0;
	}</string>
        </property>
        <property name="text">
         <string>Название задания</string>
        </property>
       </widget>
       <widget class="QPushButton" name="beginTab">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>81</width>
          <height>26</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>26</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background-color:rgb(220, 220, 220);
	border: none;
	color: rgb(40,40,40);
	text-align: left;
	padding-left: 6px;
	padding-top:2px;
	margin-top:6;
}
	QPushButton:hover
	{
		background-color:rgb(255, 220, 220);
		margin-top:0;
	}
	</string>
        </property>
        <property name="text">
         <string>Начало</string>
        </property>
       </widget>
       <widget class="QPushButton" name="courseTab">
        <property name="geometry">
         <rect>
          <x>81</x>
          <y>0</y>
          <width>465</width>
          <height>26</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>26</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	border: none;
	background-color:rgb(200, 200, 200);
	color: rgb(40,40,40);
	text-align: left;
	padding-left: 6px;
	padding-top:2px;
	margin-top:6;
}
	QPushButton:hover
	{
		background-color:rgb(220, 220, 220);
	margin-top:0;
	}</string>
        </property>
        <property name="text">
         <string>Название курса</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="list">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Roboto</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QListWidget
{
	background-color: rgb(230, 230, 230);
	alternate-background-color: rgb(220, 220, 220);
}
QListWidget:item:hover
{
	
}
QScrollBar:vertical{
			background:rgb(210, 210, 210);
            border: none;
            width:16px;    
            margin: 0px 0px 0px 0px;
        }
        QScrollBar:handle:vertical{         
       
            min-height: 0px;
          	border: 0px solid red;
			border-radius: 4px;
			margin: 0px 4px 0px 4px;
			background-color: rgb(180, 180, 180);
        }
	    QScrollBar:handle:vertical:hover {         
       
            min-height: 0px;
          	border: 0px solid red;
			border-radius: 4px;
			margin: 0px 3px 0px 3px;
			background-color: rgb(160, 160, 160);
        }
        QScrollBar::add-line {       
            height: 0px;           
        }
        QScrollBar::sub-line {
            height: 0 px;      
        }
QScrollBar:horizontal{
			background:rgb(210, 210, 210);
            border: none;
            height:16px;    
            margin: 0px 0px 0px 0px;
        }
        QScrollBar:handle:horizontal{         
       
            min-height: 0px;
          	border: 0px solid red;
			border-radius: 4px;
			margin: 4px 0px 4px 0px;
			background-color: rgb(180, 180, 180);
        }
	    QScrollBar:handle:horizontal:hover {         
       
            min-height: 0px;
          	border: 0px solid red;
			border-radius: 4px;
			margin: 3px 0px 3px 0px;
			background-color: rgb(160, 160, 160);
        }

QScrollBar::sub-page, QScrollBar::add-page
 {
    background: rgba(0,0,0,0);
}
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="resizeMode">
        <enum>QListView::Fixed</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="FilterLay">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="filterEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>31</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit
{
	color: rgb(40, 40, 40);
	background-color: rgb(220, 220, 220);
	border:none;
}

QLineEdit::hover
{
	color: rgb(40, 40, 40);
	background-color: rgb(210, 210, 210);
	border:none;
}</string>
         </property>
         <property name="maxLength">
          <number>64</number>
         </property>
         <property name="placeholderText">
          <string>Поиск</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveFilterButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Сохранить настройки фильтра поиска</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border:none;
	background-color: rgb(220, 220, 220);


}
	QPushButton:hover
	{
		background-color: rgb(210, 210, 210);
	}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
